homeassistant/components/zwave_js/light.py

```diff
@@ -320,3 +323,4 @@ def _calculate_color_values(self) -> None:
         elif ww_val or cw_val:
             # only one white channel
             self._supports_white_value = True
+            # FIXME: Update self._white_value
```

Comment:
We're not updating white value state attribute.

Reply:
FIXME will be raised as a pylint error.

Reply:
We already disable it upstream I think.